pdf/creator/basic_line.go

```diff
@@ -0,0 +1,88 @@
+package creator
+
+import (
+	"github.com/unidoc/unidoc/pdf/contentstream/draw"
+	"github.com/unidoc/unidoc/pdf/model"
+	"math"
+)
+
+// BasicLine defines a line between point 1 (X1,Y1) and point 2 (X2,Y2).  The line ending styles can be none (regular line),
+// or arrows at either end.  The line also has a specified width, color and opacity.
+// Implements the Drawable interface and can be drawn on PDF using the Creator.
+type BasicLine struct {
```

Comment:
Let's remove BasicLine from creator. It was useful for testing purposes, but I think it is confusing from the end user perspective to have both creator.Line and creator.BasicLine.  It is also not used within the creator package (using draw.BasicLine directly).